ecosystem/typescript/sdk/src/aptos_types/transaction.ts

```diff
@@ -381,15 +383,24 @@ export class MultiAgentRawTransaction extends RawTransactionWithData {
   constructor(
     public readonly raw_txn: RawTransaction,
     public readonly secondary_signer_addresses: Seq<AccountAddress>,
+    public readonly fee_payer_address?: AccountAddress,
```

Comment:
Can we separate this out into a different function (set Gas fee payer) to make this flow a lot more explicit? Users should be aware that setting a separate gas payer would require a different signature

Reply:
Done